[New Check]: TimeSeries rate is zero (#404)
NeurodataWithoutBorders/nwbinspector: docs/best_practices/time_series.rst

If the ``resolution`` of a :ref:`nwb-schema:sec-TimeSeries` is unknown, use ``-1.0`` or ``NaN`` to indicate this.

Check function: :py::meth:`~nwbinspector.checks.time_series.check_resolution`

.. _best_practice_non_zero_rate:

> Zero Rate
> ~~~~~~~~~~~~~~~~~~

Reviewer: ```suggestion
Zero Rate
~~~~~~~~~
```

length of tildes should match length of title; sphinx build will warn about this but I don't think it causes an error yet

> If ``data`` field of :ref:`nwb-schema:sec-TimeSeries` has more than one frame, thus first dimension that represent time is
> greater than one, ``rate`` should not be 0.0.

Reviewer: ```suggestion
If the ``data`` field of :ref:`nwb-schema:sec-TimeSeries` has more than one frame, and according to :ref:`best_practice_data_orientation` this axis ought to be time, then the ``rate`` field should not be ``0.0``.
```
